CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
+* Add support for "thawing" objects. `Realm`, `Results`, `List` and `Object` now
+have `thaw()` methods which return a live copy of the frozen object. This enables app behvaior where a frozen object can be made live again in order to mutate values. For example, first freezing an object passed into UI view, then thawing the object in the view to update values.
+
+### Fixed
+* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-cocoa/issues/????), since v?.?.?)
+* None.
```

Comment:
Changelog merge was incorrect.

Reply:
Removed empty ###fixed and incorrect internal body.